common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -87,6 +87,35 @@ Here is an example from the same 5" quad with the harmonic notch configured:
     :target:  ../_images/post-tune-fft.png
     :width: 450px
 
+Step by step guide for Throttle-based Setup
+===========================================
+Based on: flat octocopter, Cube Orange, 6s Batteries, 16" Propellers, copter weight around 9kg.
+
+1. Check the parameters in Mission Planner and change them to:
+- Set :ref:`INS_LOG_BAT_MASK <INS_LOG_BAT_MASK>` = 1
+- Set :ref:`LOG_BITMASK <LOG_BITMASK>` (without RAW IMU)
+- Set :ref:`INS_HNTCH_ENABLE <INS_HNTCH_ENABLE>` = 1
+- Set :ref:`INS_LOG_BAT_MASK <INS_LOG_BAT_MASK>` = 7
+- Set :ref:`INS_LOG_BAT_OPT <INS_LOG_BAT_OPT>` = 1
+- Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1
+
+2. Do the gentle hover flight at least for 30 seconds.
+
+3. Analyze the flight log in Mission Planner (Cntrl+F) FFT option and select new DF log (load your .bin file log from this gentle flight)
+
+4. Check the value of first high peak and remember the value. In my case it was 70Hz
```

Comment:
Best to use the first peak (not necessarily the highest). You can tell whether you found the first harmonic because all the subsequent peaks should be a multiple of this value.